client/src/pages/Job/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, Tab, Tabs, Typography, useMediaQuery } from "@mui/material";
+import { useEffect, useState } from "react";
+import { Info, Event, Contacts, Description } from '@mui/icons-material';
+
+export default function Job() {
+    // Constants
+    const BASE_CLEARBIT_URL = 'https://logo.clearbit.com/';
+
+    // State variables
+    const [currTab, setCurrTab] = useState(0);
+    const [role, setRole] = useState("Software Engineer");
+    const [company, setCompany] = useState("Google");
+    const [url, setUrl] = useState("google.com");
```

Comment:
Is there a reason to set these initially?

Reply:
Just to give them clear types like string or number

Reply:
But no, they can be generic